apps/hash-api/src/graph/knowledge/system-types/linear-integration-entity.ts

```diff
@@ -349,24 +352,79 @@ export const linkIntegrationToWorkspace: ImpureGraphFunction<
       ],
     });
   } else {
-    await createLinkEntity<SyncLinearDataWith>(context, authentication, {
-      webId: extractWebIdFromEntityId(linearIntegrationEntityId),
-      properties,
-      linkData: {
-        leftEntityId: linearIntegrationEntityId,
-        rightEntityId: workspaceEntityId,
+    /**
+     * Allow the user creating the link, the Linear bot and the web admins to administer the link (e.g. to delete it),
+     * and allow other web members to view the link.
+     */
+    const linearIntegrationRelationships: EntityRelationAndSubjectBranded[] = [
+      {
+        relation: "administrator",
+        subject: {
+          kind: "account",
+          subjectId: authentication.actorId,
+        },
       },
-      entityTypeIds: [
-        systemLinkEntityTypes.syncLinearDataWith.linkEntityTypeId,
-      ],
-      relationships: [
-        ...createDefaultAuthorizationRelationships(authentication),
-        {
-          // Allow the system account ID to view the link
-          relation: "viewer",
-          subject: { kind: "account", subjectId: systemAccountId },
+      {
+        relation: "administrator",
+        subject: {
+          kind: "account",
+          subjectId: linearBotMachineId,
         },
-      ],
+      },
+      {
+        relation: "setting",
+        subject: {
+          kind: "setting",
+          subjectId: "administratorFromWeb",
+        },
+      },
+      {
+        relation: "setting",
+        subject: {
+          kind: "setting",
+          subjectId: "viewFromWeb",
+        },
+      },
+    ];
+
+    const linearIntegrationWebId = extractWebIdFromEntityId(
+      linearIntegrationEntityId,
+    );
+
+    const linkEntity = await createLinkEntity<SyncLinearDataWith>(
+      context,
+      authentication,
+      {
+        webId: linearIntegrationWebId,
+        properties,
+        linkData: {
+          leftEntityId: linearIntegrationEntityId,
+          rightEntityId: webEntityId,
+        },
+        entityTypeIds: [
+          systemLinkEntityTypes.syncLinearDataWith.linkEntityTypeId,
+        ],
+        relationships: linearIntegrationRelationships,
+      },
+    );
+
+    const linkEntityUuid = extractEntityUuidFromEntityId(
+      linkEntity.metadata.recordId.entityId,
+    );
+
+    await createPolicy(context.graphApi, authentication, {
```

Comment:
see https://github.com/hashintel/hash/pull/7405
```suggestion
    // TODO: allow creating policies alongside entity creation
    //   see https://linear.app/hash/issue/H-4622/allow-creating-policies-alongside-entity-creation
    await createPolicy(context.graphApi, authentication, {
```